Automattic/jetpack
# Carousel: add new attachment ID filter

Fixes #32718

## Proposed changes:

It should allow third-parties access to that data if necessary.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

Not much to test here, you could check that the Carousel feature still works:

1. Go to Jetpack > Settings and enable the Carousel feature.
2. Go to Posts > Add New and insert a gallery.
3. Add some images, with some metadata (title, description, ...).
4. Publish your post.
5. When viewed on the frontend, you should be able to click on the image to open the modal view and see the metadata you had added.
